src/utils/colonyMotions.ts

```diff
@@ -184,18 +185,18 @@ export interface MotionValue {
 }
 
 export const getUpdatedDecodedMotionRoles = (
-  colony: Colony,
   recipient: AnyUser,
   fromDomain: number,
-  roles: ActionUserRoles[],
+  currentRoles: ColonyRoles = [],
+  setRoles: ActionUserRoles[],
```

Comment:
what "setRoles" means? when i see set prefix i usually think of setter methods

Reply:
but its just nitpick :P

Reply:
So we have a prop that's coming from the contracts that is called `setTo`, which can be `true` or `false`

Roles that are inside the `setRoles` array, are roles which have that prop.

That's where the naming comes from :)